core/src/main/java/org/hibernate/ogm/dialect/eventstate/impl/EventContextManagerInitiator.java

```diff
@@ -6,18 +6,19 @@
  */
 package org.hibernate.ogm.dialect.eventstate.impl;
 
-import java.util.Map;
-
-import org.hibernate.boot.registry.StandardServiceInitiator;
+import org.hibernate.boot.spi.SessionFactoryOptions;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.ogm.dialect.spi.EventContextManagerAwareGridDialect;
+import org.hibernate.ogm.dialect.spi.GridDialect;
 import org.hibernate.service.spi.ServiceRegistryImplementor;
+import org.hibernate.service.spi.SessionFactoryServiceInitiator;
 
 /**
  * Contributes the {@link EventContextManager} service.
  *
  * @author Gunnar Morling
  */
-@SuppressWarnings("rawtypes")
-public class EventContextManagerInitiator implements StandardServiceInitiator<EventContextManager> {
+public class EventContextManagerInitiator implements SessionFactoryServiceInitiator<EventContextManager> {
```

Comment:
Mmmh, probably because I thought I would need a SessionFactoryServiceRegistry as the old isEventContextRequired method was using it.

But, it looks like it's useless.

If you're OK with the general approach, I'll withdraw that part.